File: TruffleCommon/src/main/java/io/synople/truffle/common/model/Vendor.java
package io.synople.truffle.common.model;

import java.util.List;

public class Vendor {

    private String id;
    private String name;
    private List<Item> items;

    public Vendor(String id, String name, List<Item> items) {
        this.id = id;
        this.name = name;
        this.items = items;
    }
}

File: TruffleCommon/src/main/java/io/synople/truffle/common/model/Vendor.kt
package io.synople.truffle.common.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Vendor(
    var id: String = "",
    var name: String = "",
    var items: MutableList<Item> = mutableListOf()
) : Parcelable
